catalogue/webapp/pages/item.tsx

```diff
@@ -439,9 +440,18 @@ export const getServerSideProps: GetServerSideProps<
   )?.url;
   const iiifImageLocation = getDigitalLocationOfType(work, 'iiif-image');
 
-  const manifestOrCollection = iiifPresentationUrl
-    ? await fetchJson(iiifPresentationUrl)
-    : undefined;
+  let manifestOrCollection;
+  if (globalContextData.toggles.switchIIIFManifestSource) {
+    const iiifPresentationUrlOnStage =
+      iiifPresentationUrl && convertPresentationUrlToStage(iiifPresentationUrl);
+    manifestOrCollection = iiifPresentationUrlOnStage
+      ? await fetchJson(iiifPresentationUrlOnStage)
+      : undefined;
+  } else {
+    manifestOrCollection = iiifPresentationUrl
+      ? await fetchJson(iiifPresentationUrl)
+      : undefined;
```

Comment:
Could this be pulled out into a generalised method so if we need to change it we don't forget to do it in other places?